x-pack/plugins/case/server/routes/api/update_case.ts

```diff
@@ -5,9 +5,16 @@
  */
```

Comment:
Just out of curious, would case have it's own entry with the same level as SIEM as well in the future?

Reply:
thats the plan :)